### DISCUSSION TITLE
Missing columns

### DISCUSSION THREAD

**Discussion by gooly** [uBlockOrigin/uBlock-issues#1926] [General] [answered]
Hi, 
I would like to enable on https://www.heise.de/news/Sicherheitsupdates-vom-Januar-2022-mit-massiven-Kollateralschaeden-in-Windows-6325265.html this: responder.wt.heise.de but my uBlock 1.40.8 only has the colored bars on the left side (in my case red), the names like responder.wt.heise.de and then right away the column with ++,+, -
How do I allow responder.wt.heise.de generally?
In the docs about uBlock 1.40.8 are two more columns mentioned. Unfortunately I can't upload an image her but here you can watch it: https://we.tl/t-y2jmiImdie

**Comment by ghost** [comment]
https://github.com/gorhill/uBlock/wiki/Dynamic-filtering

**Comment by gorhill** [comment]
[Advanced user features](https://github.com/gorhill/uBlock/wiki/Advanced-user-features#dynamic-filtering). Because of the fact that you didn't remember how you enabled advanced users features, I will reiterate to heed the warning on this page:

> If you use advanced features without fully understanding them, uBlock Origin:
> 
> <ul>
> <li>might</li>
> <li><i>probably <b>will</b></i>
> </ul>
> 
> ... behave in ways unexpected to you, and most likely you will end up being **LESS** protected than you would otherwise be with just default settings.
> 
> #### DO NOT UNDERESTIMATE THIS WARNING.

**Comment by gooly** [comment]
Ok I finally found and set the advanced user but still I can't get rid of the blocking of responder.wt.heise.de of 
https://www.heise.de/news/Sicherheitsupdates-vom-Januar-2022-mit-massiven-Kollateralschaeden-in-Windows-6325265.html

**Comment by gorhill** [comment, marked as answer]
> but still I can't get rid of the blocking of `responder.wt.heise.de`

It's because you are not reading the documentation carefully. You have to create an [_allow_ rule](https://github.com/gorhill/uBlock/wiki/Dynamic-filtering:-quick-guide#allow-rules) to override a block filter from static filter lists. I suggest you leave dynamic filtering alone and just use an exception filter to override the block filter:

![Screenshot from 2022-01-13 08-56-12](https://user-images.githubusercontent.com/585534/149343341-06285a90-9e0d-458a-9032-3b8405c3add7.png)

**Comment by gooly** [comment]
Ok, thanks a lot - now I have got it!!
Thanks again and all the best for 2022

**Comment by gooly** [comment]
Many documentations of devices, programs etc. are long and structured according to the principle, it consists of this and that and also that, and it can do this and that and that too. Then there is the solution to a user problem somewhere in the middle on half a page as a small part of a larger topic. The user, however, partly does not know the technical terms used. He can then assign his goal of changing something very specific neither to a topic nor to the specific technical words.
To make it short the manufacturer presents its product, the user would need however usually only an appendix with: **How do I** turn this on, or **how do I** change this and so on.
In my case: a link/script/.. is blocked, **how do I** allow it once (for testing) or permanently.
Then also immediately the reverse: **how do I** lock a link/script/.. once (for test) or permanently.
Please don't be offended I appreciated uBlock and your help very much and maybe my remark can make uBlock more user friendly
